src/components/Avatar.tsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components'
 
 export const Avatar = styled.img<{ size?: number }>`
   display: flex;
+  justify-self: end;
```

Comment:
This is a good fix, but we don't want all `Avatar` components to be aligned like this. The component can be used anywhere, not only in the `Quote`.

I suggest you to wrap `Avatar` in a `Box` component, and assign `justify` prop on the Box instead, in the `src/components/Quote`:

```js
<Box as={Avatar} justify="flex-end">
```